lib/packages.cf

```diff
@@ -94,6 +94,28 @@ body package_module yum
     #default_options => {};
 }
 
+body package_module freebsd_ports
+# @brief Define details used when interfacing with the freebsd ports package
+# module.
+#
+# **Note:** Ports are expected to be setup prior to trying to use the packages
+# promise. You may need to ensure that `portsnap extract` has been run.
+#
+# **Example:**
+# ```cf3
+# bundle agent main
+# {
+#   packages:
+#     freebsd::
```

Comment:
`freebsd_ports` is most appropriate; as pointed out, OpenBSD and OSX (also NetBSD) use "ports". `freebsd` is inappropriate due to ports vs `pkgng` vs `portmaster` vs `portshaker`+`portmaster` vs `pkgng+poudriere`.